types/tedious/index.d.ts

```diff
@@ -813,3 +813,17 @@ export declare class RequestError extends Error {
     public message: string;
     public code: string;
 }
+/**
+ * Error emitted by MSSQL Request event
+ */
+export declare class MSSQLRequestError extends Error {
```

Comment:
I don't see this class defined in `tedious`: https://github.com/tediousjs/tedious/search?q=MSSQLRequestError

Where can I find this error? Is it actually a class that exists at runtime?